### DISCUSSION TITLE
What is the license for Errata data?

### DISCUSSION THREAD

**Discussion by pombredanne** [rocky-linux/peridot#193] [Q&A] [answered]
### Is this feature request related to a problem? If so, please describe it.

What is the license for Errata data? I could not find one.
I reckon that several errata may be mirrored from RH, so their license (whatever it is) would likely apply too?

### Describe the solution you'd like to see

I need a clear open license for inclusion in https://github.com/nexb/vulnerablecode and https://github.com/nexB/vulnerablecode/issues/753

### Version and Build Information

N/A

### Additional Context

N/A

**Comment by pombredanne** [comment]
I likely filed this issue in the wrong repo .... Should be in https://github.com/resf/distro-tools/tree/main/apollo instead?

**Comment by NeilHanlon** [comment, marked as answer]
Hey - No worries about filing it here. The license for the errata data is the same as RH, that is, `CC-BY-4.0`.

> The data resources linked on this page as well as their alternative representations available through the [Security Data API](https://access.redhat.com/documentation/en-us/red_hat_security_data_api/1.0/html/red_hat_security_data_api/index) are licensed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). If you distribute this content or a modified version of it, you must provide attribution to Red Hat, Inc. and provide a link to the original.

https://access.redhat.com/security/data

**Reply by pombredanne** [reply]
Thank you ++
